gateway/ln-gateway/src/lnrpc_client.rs

```diff
@@ -220,6 +228,20 @@ impl ILnRpcClient for NetworkLnRpcClient {
         })?;
         Ok(res.into_inner())
     }
+
+    async fn update_fedimint_scids(
+        &self,
+        scids: Vec<u64>,
+    ) -> Result<EmptyResponse, LightningRpcError> {
+        let mut client = Self::connect(self.connection_url.clone()).await?;
+        let res = client
+            .update_fedimint_scids(UpdateFedimintScidsRequest { scids })
+            .await
+            .map_err(|status| LightningRpcError::FailedToCompleteHtlc {
```

Comment:
Use a different error here rather than `FailedToCompleteHtlc`

Reply:
updated to `FailedToUpdateScids`